core/src/apps/bitcoin/sign_tx/decred.py

```diff
@@ -208,6 +232,54 @@ def write_tx_output(
             writers.write_uint16(w, DECRED_SCRIPT_VERSION)
         writers.write_bytes_prefixed(w, script_pubkey)
 
+    def validate_sstx_submission(self, txo: TxOutput) -> bytearray:
```

Comment:
if these functions return the scripts, they should not be called `validate_*`.

`process_*` would be more appropriate. I'm wondering though, whether it's even worth having the sstx_submission and sstx_change handled in their own functions? there is no functionality to speak of

Reply:
https://github.com/trezor/trezor-firmware/pull/1249/commits/20aa862004cd9731524b43026be377223078237c